docs/rules/no-multiple-actions-in-effects.md

```diff
@@ -1 +1,76 @@
 # no-multiple-actions-in-effects
+
+> This rule disallows mapping to multiple actions in a single Effect.
+
+## Rule Details
+
+An Effect should map one event (action) to a single other action.
+An action can result in several other Effects being triggered, or multiple changes in the reducer, in which case the developer might be tempted to map an Effect to several actions. A more understandable approach is to dispatch one "combining" action that describes what happened, rather than multiple actions.
```

Comment:
```suggestion
An action can result in several other Effects being triggered, or multiple changes in the reducer, in which case the developer might be tempted to map an Effect to several actions. A more understandable approach is to dispatch one "combining" action that describes what happened (an event), rather than multiple actions.
```

Reply:
Agreed, this better describes what exactly an action is